algorithms/CPlusPlus/Maths/fibonaccci-series.cpp

```diff
@@ -0,0 +1,65 @@
+
+
+#include <iostream>
+
+using namespace std;
+
+/**
+ *   The Fibonacci Sequence is the series of numbers:
+ *
+ *   0, 1, 1, 2, 3, 5, 8, 13, 21, 34, ...
+ *
+ *   The next number is found by adding up the two numbers before it:, next number is the sum of previous two numbers
+ *   for example 0,
+ *   1, 1, 2, 3, 5, 8, 13, 21 etc. The first two numbers of fibonacci series are 0 and 1.
+ *
+ * */
+
+class Fibonaccci {
+public:
+
+    void print_fibonaccci_no_R() {
```

Comment:
```suggestion
    void print_fibonacci_no_R() {
```
This function has no calling function remove this one or call it from main by removing the comment // from main function